rock_paper_scissors.py

```diff
@@ -150,8 +150,8 @@ def main():
     game.rounds = int(input('How many odd rounds would you like to play? '))
 
     while not game.is_complete:
-        user.choice = input('Choose rock, paper, or scissors: ')
-        computer.choice = random.choice(VALID_CHOICES)
+        user.choice = input('Choose from the following: {}: '.format(', '.join(game.valid_choices)))
```

Comment:
Actually, since it's changing in _this_ PR I think we should attempt to address it on this branch before kicking it down the road to another issue :)

We can work on this together if you'd like. A few facts to consider about breaking down this problem (I know it comes across as tricky; googling "python oxford comma" reinforces this sentiment):
- All items need to be separated by a comma.  You can think of the very last item as just happening to contain the word "or"
- Only the last item needs to be prefixed with "or "

Keeping that in mind, we can then divide the `game.valid_choices` list into two chunks:
- The head, which contains all the items except for the last item
- The tail, which needs to start with "or "

Again, I'm happy to pair up on this, so let me know if that's what you'd like to do.